src/components/sbb-form-field/sbb-form-field.scss

```diff
@@ -165,7 +165,8 @@
   margin-block-end: var(--sbb-form-field-label-to-input-gap);
   color: var(--sbb-form-field-label-color);
 
-  :host([data-input-type='select']) & {
+  :host([data-input-type='select']) &,
```

Comment:
Can you check if floating labels work for sbb-select as for native select? (they are very new, maybe you need to rebase/merge)